mixer2: {
  Alpha Injection: /usr/lib/frei0r-1/alphainjection.so,
  Composition: /usr/lib/frei0r-1/composition.so,
  UV Map: /usr/lib/frei0r-1/uvmap.so,
  addition: /usr/lib/frei0r-1/addition.so,
  addition_alpha: /usr/lib/frei0r-1/addition_alpha.so,
  alphaatop: /usr/lib/frei0r-1/alphaatop.so,
  alphain: /usr/lib/frei0r-1/alphain.so,
  alphaout: /usr/lib/frei0r-1/alphaout.so,
  alphaover: /usr/lib/frei0r-1/alphaover.so,
  alphaxor: /usr/lib/frei0r-1/alphaxor.so,
  blend: /usr/lib/frei0r-1/blend.so,
  burn: /usr/lib/frei0r-1/burn.so,
  cairoaffineblend: /usr/lib/frei0r-1/cairoaffineblend.so,
  cairoblend: /usr/lib/frei0r-1/cairoblend.so,
  color_only: /usr/lib/frei0r-1/color_only.so,
  darken: /usr/lib/frei0r-1/darken.so,
  difference: /usr/lib/frei0r-1/difference.so,
  divide: /usr/lib/frei0r-1/divide.so,
  dodge: /usr/lib/frei0r-1/dodge.so,
  grain_extract: /usr/lib/frei0r-1/grain_extract.so,
  grain_merge: /usr/lib/frei0r-1/grain_merge.so,
  hardlight: /usr/lib/frei0r-1/hardlight.so,
  hue: /usr/lib/frei0r-1/hue.so,
  lighten: /usr/lib/frei0r-1/lighten.so,
  multiply: /usr/lib/frei0r-1/multiply.so,
  none: none,
  overlay: /usr/lib/frei0r-1/overlay.so,
  saturation: /usr/lib/frei0r-1/saturation.so,
  screen: /usr/lib/frei0r-1/screen.so,
  softlight: /usr/lib/frei0r-1/softlight.so,
  subtract: /usr/lib/frei0r-1/subtract.so,
  value: /usr/lib/frei0r-1/value.so,
  xfade0r: /usr/lib/frei0r-1/xfade0r.so,
}
